packages/venia-concept/venia-upward.yml

```diff
@@ -52,13 +99,20 @@ imageOpt:
             path: request.url.query.url
           template:
             when:
+              # Redirect to the Fastly service, which is presumed to be part
+              # the URL query, when the USE_FASTLY environment value is present
```

Comment:
I think this environment variable is no longer necessary. See [here](https://github.com/magento-research/pwa-studio/commit/1f707406eab90756ce7a22824c06af7a20caa5e6). I don't quite understand how it would be used now. @tjwiebell may have input.

Reply:
`USE_FASTLY` is no longer in use. We should probably merge in mainline and update the docs; more along the lines of:

> Fastly Image Optimization parameters are preemptively added to proxied media requests to Magento, and will be handled by Fastly, if enabled. Otherwise, requests will fall through to Magento, those parameters will be ignored, and unoptimized images will be served.

Reply:
I wanna bring it back, so we know when to base media URLs directly on the instance.